doc/tests.rst

```diff
@@ -0,0 +1,156 @@
+Tests
+======
+
+As EasyAdmin is part, functionally testing the admin pages can leverage the
```

Comment:
Je suis pas sûr de comprendre la phrase. J'ai l'impression que t'as oublié un bout après "is part"... mais c'est peut-être mon anglais

Reply:
yep... je ne sais même plus ce que je voulais dire.. ca y est j'ai retrouvé... et "part" n'en fait pas partie :D